### PR TITLE
Remove static httpClient  since it is disposed every time that the cl…

### PR DESCRIPTION
**Pull request by r0lan2**
I can see that Channel class is using a static reference to httpclient. That should be good if you want to decrease the number of "open connection" to your endPoint. But the strange thing  that I can see here is that you are disposing your httpClient for every call to RequestAsync, what is exactly don't should be done for "static" strategy.
In order don't get issues related Thread safety I prefer to have the instace inside of the method RequestAsync. 
Please read this article where you can see more about how to use Httpclient.
https://medium.com/@nuno.caneco/c-httpclient-should-not-be-disposed-or-should-it-45d2a8f568bc

In summary the article talk about if you should use httpclient as a static member or create instace of this class for each call.

### CODE REVIEW

**Comment by r0lan2** [comment]
I am not expert with github. What shall I do to with my PR ?
Update my fork and ..?
any guide could help me :)

**Comment by afiebig** [comment]
Hi @r0lan2 you need to merge changes from master, and resolve the conflicts. The Channel class is no longer working as Async because it was giving problems wit MVC projects. 

You should also run test locally before you push to github. Also, you can check te result from te build in semaphore clicking in the details link under the Checks section.

File: Transbank/Onepay/Net/Channel.cs [hunk 1]
```diff
@@ -6,45 +6,42 @@
 
 namespace Transbank.Onepay.Net
 {
-    public abstract class Channel
+      public abstract class Channel
```
Comment (afiebig) [inline Transbank/Onepay/Net/Channel.cs]: You have an extra space here.

File: Transbank/Onepay/Net/Channel.cs [hunk 2]
```diff
@@ -6,45 +6,42 @@
...
+                    response.EnsureSuccessStatusCode();
+                    var jsonResponse = response.Content.ReadAsStringAsync().ConfigureAwait(false).GetAwaiter().GetResult();
+                    return jsonResponse;
+            }            
         }
 
-        internal static HttpClient Client { get; private set; }
 
-        public static string PostString(string url, string RequestMethod, string Query )
+
```
Comment (afiebig) [inline Transbank/Onepay/Net/Channel.cs]: And an extra line here.

File: Transbank/Onepay/Net/Channel.cs (hunk 1 above)
Comment (afiebig) [reply Transbank/Onepay/Net/Channel.cs]: 1. For catchig this mistakes I use git diff  from the cli, but maybe VisualStudio has a built in diff tool.
2. There is a Test Project alongside the SDK, you can run it locally using `dotnet test TransbankTest` or using the test explorer in VS.
3. Travis runs on each PUSH and PR (also Travis is a WIP) you have to look for SemaphoreCI wich also runs on every PUSH and PR. 

This will be son clearer inside the README, sorry for the delay, we are still working on the documentation.

**Comment by r0lan2** [comment]
hi @afiebig I have updated my PR but still is RED :(...for me is little hard to see now where is my error. Can you help me :(

**Comment by afiebig** [comment]
@r0lan2 it is because you applied te fix to the netCorre version before i send the comment requesting the change, give me a sec and i will fix it.
